src/app/server/Server.cpp

```diff
@@ -492,4 +494,18 @@ void Server::ResumeSubscriptions()
 }
 #endif
 
+KvsPersistentStorageDelegate CommonCaseDeviceServerInitParams::sKvsPersistenStorageDelegate;
+PersistentStorageOperationalKeystore CommonCaseDeviceServerInitParams::sPersistentStorageOperationalKeystore;
+Credentials::PersistentStorageOpCertStore CommonCaseDeviceServerInitParams::sPersistentStorageOpCertStore;
+Credentials::GroupDataProviderImpl CommonCaseDeviceServerInitParams::sGroupDataProvider;
+IgnoreCertificateValidityPolicy CommonCaseDeviceServerInitParams::sDefaultCertValidityPolicy;
```

Comment:
Why do these appear here? They are static inlined in InitializeStaticResourcesBeforeServerInit

Reply:
Having static local variables in a function instead of static members gives code-size penalty, so I moved them to save a bit.